recipes/mdsapt/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "mdsapt" %}
+{% set version = "2.0.3_rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/calpolyccg/MDSAPT/archive/v2.0.3-rc1.tar.gz
+  sha256: 4d90626d9494a8531d9190c5d5b41a9e33fcbc11f6138336494b402fa59dd450
+
+build:
+  skip: True  # [win or py < 39]
```

Comment:
What does it mean that it isn't supported on Windows? If a dependency is missing and CI is failing because of that, that's OK. If the actual code isn't working, then you should add a run dependency on `__unix`

Reply:
The project doesn't support Windows because its transitive dependencies don't, and it's not tested or built on Windows either. The CI fails on Windows if I get rid of the selector, anyways.

Reply:
This then sounds like a case for making this a noarch: python package. Tge failing CI is OK. This means it is not installable on Windows but we still only have a single build. Thisnis is the preferred way to solve the issue

Reply:
Oh, alright then. I guess I'll do that